File: backend/ViewModels/UserBookingsViewModel.cs
﻿using BookingApp.Enums;
using BookingApp.Models;

namespace BookingApp.ViewModels
{
    public class UserBookingsViewModel
    {   
        public List<UserBookingViewModel> ParkingsVm { get; set; }  
        public List<UserBookingViewModel> BookingsVm { get; set; }
        public List<BookingPlaceWithBookingsViewModel> FixedPlacesVm { get; set; }
        public List<BookingPlaceWithBookingsViewModel> FixedParkingsVm { get; set; }
        public UserViewModel? BookedByUserVm { get; set; }
    }
}

File: frontend/src/app/api-generated/models/user-bookings-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { BookingPlaceViewModel } from '../models/booking-place-view-model';
import { UserBookingViewModel } from '../models/user-booking-view-model';
import { UserViewModel } from '../models/user-view-model';
export interface UserBookingsViewModel {
  bookedByUserVm?: UserViewModel;
  bookingsVm?: Array<UserBookingViewModel> | null;
  fixedParkingsVm?: Array<BookingPlaceViewModel> | null;
  fixedPlacesVm?: Array<BookingPlaceViewModel> | null;
  parkingsVm?: Array<UserBookingViewModel> | null;
}
